src/main/java/frc/trigon/robot/subsystems/collector/CollectorSubsystem.java

```diff
@@ -0,0 +1,41 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class CollectorSubsystem extends SubsystemBase {
+    private final static CollectorSubsystem INSTANCE = new CollectorSubsystem();
+    private final TalonSRX motor = CollectorConstants.MOTOR;
+    // motor.getSupplyCurrent
+
+    public static CollectorSubsystem getInstance() {
+        return INSTANCE;
+    }
+    
+    private CollectorSubsystem() {
+    }
+
+    /**
+     * creates a command that gives power to the motor according to the inputted state.
+     * @param state the inputted state
+     * @return the command
+     */
+    public CommandBase setCollectorStateCommand(CollectorConstants.CollectorStates state){
+        return new FunctionalCommand(
+                () -> {},
+                () -> motor.set(ControlMode.PercentOutput, state.power),
```

Comment:
This can and should be only on initializing. 
Also, extract this into a method called `setTargetState`

Reply:
wdym?

Reply:
Current, how you did it, is that it will set the target power every command period. Instead, you could only do it when the command initializes, and even make it a StartEndCommand